site/cypress/e2e/Cypress-Feature/forums.spec.js

```diff
@@ -40,10 +39,11 @@ const upduckPost = (thread_title) => {
 
 const checkStatsUpducks = (fullName, numUpducks) => {
     // Check the stats page for a user with fullName and
-    // number of upducks numUpducks
+    // at least number of upducks numUpducks
     cy.get('[data-testid="more-dropdown"]').click();
     cy.get('#forum_stats').click();
-    cy.get('[data-testid="user-stat"]').contains(fullName).siblings('[data-testid="upduck-stat"]').should('contain.text', numUpducks);
+    cy.get('[data-testid="user-stat"]').contains(fullName).siblings('[data-testid="upduck-stat"]')
+        .invoke('text').then(parseInt).should('be.gte', numUpducks);
```

Comment:
I don't think we should change the stats, making sure that it is 3 is good. I think that the reason that it fails with a upduck value of 8 is because the previous one messes with the later one

Reply:
I think I agree, changed